pkg/internal/export/otel/metrics.go

```diff
@@ -322,6 +329,10 @@ func (mr *MetricsReporter) metricAttributes(span *request.Span) attribute.Set {
 			attrs = append(attrs, semconv.NetSockPeerName(span.Host))
 			attrs = append(attrs, semconv.NetSockPeerPort(span.HostPort))
 		}
+	case request.EventTypeSQLClient:
```

Comment:
Sorry I didn't notice this before, but I think we should not use the `span.Path`, this will lead to cardinality explosion. What we are setting is a table name, which on certain systems can change often, e.g. folks may have daily tables, hourly tables sometimes.

I think it's best we put the DBOperation instead, which is much lower cardinality, in your case `span.Method`. Along these lines, I also don't think the table name would be very useful without the operation.